src/videohub/README.md

```diff
@@ -0,0 +1,2 @@
+The files within this folder are all manually copied from:
+https://github.com/bitfocus/companion-module-bmd-videohub/
```

Comment:
Do you think this should be a git submodule or something fancy, or is this fine?